src/js/shield_defs.js

```diff
@@ -1424,6 +1424,11 @@ export function loadShields() {
 
   // Mississippi
   shields["US:MS"] = ovalShield(Color.shields.white, Color.shields.black);
+  shields["US:MS:Scenic"] = banneredShield(
+    ovalShield(Color.shields.white, Color.shields.blue),
+    ["SCEN"],
```

Comment:
The banner is also appropriate because there’s always a “SCENIC ROUTE” banner over the shield in the real world.

“SCEN” isn’t the most intuitive abbreviation. Maybe separately we could try to accommodate longer banner text by reducing the pitch or squishing the text horizontally. That could let us spell out “TRUCK” too.